lib/setup.js

```diff
@@ -759,19 +757,16 @@ function processCommand(command, args, params, callback) {
                 name = name.split('@')[0];
             }
 
-            if (!name) {
-                console.log('Please provide the name of the adapter to rebuild');
-                return void callback(EXIT_CODES.INVALID_ADAPTER_ID);
-            }
-
-            const rebuildCommand = params.install ? 'install' : 'rebuild';
-            installNpm(name, rebuildCommand, (err, _adapter) => {
-                if (err) {
-                    processExit(err);
-                } else {
+            console.log(`Rebuilding ${name}... (system call)`);
```

Comment:
If we do not rebuild the specific adapter we should also adjust logmessage to something like "ignore provided name "${name}" for rebuild and rebuild all for now" or such ... then we see in testing if we still need a name specific version. But such a log is easier for later debugging